lib/math.rb

```diff
@@ -0,0 +1,29 @@
+module Math
+  def average(array, round_precision)
+    (sum(array) / array.length).round(round_precision)
+  end
+
+  def average_by(total_sum, total_length, round_precision)
+    (total_sum.to_f / total_length).round(round_precision)
+  end
+
+  def sum(array)
```

Comment:
I just realized, after looking at some of the other methods you wrote, that we could just use the `sum` enumerable instead of having this helper function.

Do you think we should just nix this function and change any calls to it to just use `array.sum`?